src/rhsm/connection.py

```diff
@@ -1860,39 +1744,27 @@ def getOwnerList(self, username: str) -> List[dict]:
         owners = [x for x in (owners or []) if x is not None]
         return owners
 
-    # FIXME: It is not used by any client tool (neither virt-who)
-    def getOwnerHypervisors(self, owner_key: str, hypervisor_ids: List[str] = None) -> List[dict]:
-        """
-        If hypervisor_ids is populated, only hypervisors with those ids will be returned. This method
-        requires admin connection and authenticated using username/password
-        :param owner_key: owner ID (organization ID)
-        :param hypervisor_ids: list of string with hypervisor UUIDs
-        """
-        method = "/owners/%s/hypervisors?" % owner_key
-        for hypervisor_id in hypervisor_ids or []:
-            method += "&hypervisor_id=%s" % self.sanitize(hypervisor_id)
-        return self.conn.request_get(method, description=_("Fetching organization hypervisors"))
-
-    # FIXME: this method should return result of REST API call. Candlepin server response with
-    # code 204 and no content is specified. If code 204 is returned, then True should be returned
-    def unregisterConsumer(self, consumerId: str) -> None:
+    def unregisterConsumer(self, consumerId: str) -> bool:
         """
         Deletes a consumer from candlepin server
         :param consumerId: consumer UUID (it could be found in consumer cert, when system is registered)
         """
         method = "/consumers/%s" % self.sanitize(consumerId)
-        return self.conn.request_delete(method, description=_("Unregistering system"))
+        result = self.conn.request_delete(method, description=_("Unregistering system"))
+        if result is not None:
+            status = result.get("status")
+            return status == 204
+        return False
```

Comment:
It looks better, but it actually doesn't work, because `request_delete()` return only content. See my previous comment.